OpenRA.Mods.Common/Traits/Buildings/TransformsIntoAircraft.cs

```diff
@@ -123,6 +123,8 @@ void IResolveOrder.ResolveOrder(Actor self, Order order)
 
 				var targetActor = order.Target.Actor;
```

Comment:
This is currently not used. We probably should be using this to check`if (!AircraftCanEnter(targetActor)) return;`?

The `var target` in the Move case above is also not used!

Reply:
Fixed now, thanks.